nix/workbench/backend/nomad/podman.sh

```diff
@@ -0,0 +1,275 @@
+usage_nomadpodman() {
+  # Using a unique help message for all Nomad "sub-backends"
+  usage_nomad
+}
+
+backend_nomadpodman() {
+
+  op=${1:?$(usage_nomadpodman)}; shift
+
+  case "${op}" in
+
+    name )
+      # Can be:
+      # nomadpodman       (Using podman task driver in the cloud is not planned)
+      # nomadexec    (Starts Nomad agents supporting the nix_installable stanza)
+      # nomadcloud  (IOG Nomad Agents and Amazon S3 with credentials from Vault)
+      echo 'nomadpodman'
+    ;;
+
+    # Generic backend sub-commands, shared code between Nomad sub-backends.
+
+    describe-run )
+      backend_nomad describe-run         "$@"
+    ;;
+
```

Comment:
@fmaste, you can simplify this passthrough very significantly -- you can see how the top-level `wb` does it, at the end of the file, when handling aliases.

Reply:
It was simpler, the way you recommend, but I used this form to debug and I kept it